src/locale/languages/en.js

```diff
@@ -157,6 +157,9 @@ export const en = {
     followingList: {
       title: 'Following',
     },
+    followYou: {
+      title: 'Follows you',
+    },
```

Comment:
And what if you add new phrases for translation into all language files? Then it will be clear what needs to be translated in other languages. In fact, we need to do this, because we get the situation that in the English version we have `followYou`, and in that case there will be an error (missing value) in other languages.